nixos/modules/security/setuid-wrappers.nix

```diff
@@ -115,7 +119,7 @@ in
           # programs to be wrapped.
           SETUID_PATH=${config.system.path}/bin:${config.system.path}/sbin
 
-          rm -f ${wrapperDir}/* # */
+          rm -rf ${wrapperDir}/* # */
```

Comment:
Hope we're always sure `wrapperDir` is never empty :D It's always better to rm -f *, then rmdir.